packages/react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@concordium/react-components",
-    "version": "0.3.0",
+    "version": "0.4.0-rc.7",
```

Comment:
Where's the RC version coming from? Has this been published somewhere?

Reply:
Yes, for testing.